### PR TITLE
website/integrations: Update OPNsense LDAP instructions

### PR DESCRIPTION
**Pull request by Meliox**
Changes:
- Fixed a typo in OPNsense instruction for the service-account used for bind credentials
- Added additional step to clarify how to get the opnsense users setup up
- Updated versions numbers more recent/tested by me

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [X] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/integrations/services/opnsense/index.md [hunk 1]
```diff
@@ -92,6 +92,10 @@ In OPNsense, go to _System/Settings/Administration_ and under _Authentication_ a
 
 ![](./opnsense2.png)
 
+### Step 7
+You can now either import user(s) or syncronise from Authentik LDAP. See https://docs.opnsense.org/manual/how-tos/user-ldap.html for more.
```
Comment (kensternberg-authentik) [inline website/integrations/services/opnsense/index.md:96]: Misspelling: "[synchronize](https://dictionary.cambridge.org/us/dictionary/english/synchronize)." Also, @tanberry, I don't think the optional pluralization of `user(s)` is grammatically or semantically useful; I recommend going with just "users".

**Review by kensternberg-authentik — COMMENTED** [review]
A typo needs fixing, and I want a second set of in-house eyes on one word choice.

File: website/integrations/services/opnsense/index.md (hunk 1 above)
Comment (tanberry) [reply website/integrations/services/opnsense/index.md:96]: Agree, either works but let's go with "users"; it is more simple.

**Review by tanberry — CHANGES_REQUESTED** [review]
Thanks for this, @Meliox, just a small suggestion and then we can  merge.

**Comment by Meliox** [comment]
Thanks for the review. Ressolved requested changes.

**Comment by tanberry** [comment]
Hi @Meliox can you run `make website` to run the linter/prettier, please? The build is failing there...

**Comment by Meliox** [comment]
@tanberry: Sure. I did that now, I think. Awaiting build workflows report.

**Review by tanberry — APPROVED** [review]

**Pull request merged**